index.bs

```diff
@@ -4967,8 +4977,7 @@ Liaisons</h2>
 	a Memorandum of Understanding with another organization.
 	For the purposes of the W3C Process a <dfn id="mou" export lt="Memorandum of Understanding | Memoranda of Understanding | MoU">Memorandum of Understanding</dfn> (<abbr>MoU</abbr>)
 	is a formal agreement or similar contractual framework between W3C and another party or parties,
-	other than agreements between the [=Hosts=]
-	or between [=Hosts=] and W3C members
+	or between W3C and W3C members
 	for the purposes of membership
 	and agreements related to the ordinary provision of services
 	for the purposes of running W3C,
```

Comment:
MoUs are not involved in "the purposes of membership" and I think not in "agreements related to the ordinary provision of services for the purposes of running W3C".  More substantial rephrasing is needed here.  I *think* this will serve the purpose:
```suggestion
	other than between W3C and W3C members
	for the purposes of membership,
	and other agreements related to the ordinary provision of services
	for the purpose of running W3C,
```

Reply:
can we split handling MoUs into its own thread/issue/PR, separate from AB-BoD liaisons?

Reply:
Sorry, I probably caused this confusion. The commits about the MoU are included in this PR because they're in https://github.com/w3c/w3process/pull/658, and this PR has a dependency on that one (for a definition of the Board). Github doesn't have a feature to expose PRs depending on PRs other than including the first one in the second one. But the commit meant to be under review here isn't the one about the MoU, only https://github.com/w3c/w3process/pull/669/commits/785aa1656a3235e3424845233534636814382b34. I have integrated @TallTed's comment in that upstream Pull Request. If any follow up discussion on this part is needed, please take it to https://github.com/w3c/w3process/pull/658 rather than here.